crypto/fipsmodule/rsa/rsa_impl.c

```diff
@@ -366,11 +367,17 @@ static BN_BLINDING *rsa_blinding_get(RSA *rsa, unsigned *index_used,
   assert(rsa->mont_n != NULL);
 
   BN_BLINDING *ret = NULL;
+  
   const uint64_t fork_generation = CRYPTO_get_fork_generation();
+  uint32_t snapsafe_generation = 0;
+  OPENSSL_UNUSED int snapsafe_status = CRYPTO_get_snapsafe_generation(&snapsafe_generation);
+  (void) snapsafe_status; // In MSVC, |OPENSSL_UNUSED| is not enough.
```

Comment:
Why don't we check the status?

Reply:
Because here we just do a best effort: not caring if either fork or snapsafe detection is not available.